klayout/drc/rule_decks/tail.drc

```diff
@@ -14,10 +14,6 @@
 # limitations under the License.
 ################################################################################################
 
-
-
 exec_end_time = Time.now
 run_time = exec_end_time - exec_start_time
-logger.info("%s DRC Total Run time %f seconds" % [table_name, run_time])
-
-
+logger.info(format('%<table>s DRC Total Run time %<time>f seconds', table: table_name, time: run_time))
```

Comment:
any reason you don't use ruby string interpolation here?
see https://docs.ruby-lang.org/en/3.0/syntax/literals_rdoc.html#label-Strings

Reply:
Done